osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -215,6 +215,12 @@ public BeatmapCarousel()
             InternalChild = new OsuContextMenuContainer
             {
                 RelativeSizeAxes = Axes.Both,
+                Padding = new MarginPadding
+                {
+                    // Avoid clash between scrollbar and osu! logo.
+                    Top = 10,
+                    Bottom = 100,
```

Comment:
While this does fix the scrollbar it also changes the scrollable bounds of the carousel in comparison to everything else on the screen.

| | master | this PR |
| :-: | :-: | :-: |
| scrolled maximally to start | ![osu_2024-01-25_09-59-41](https://github.com/ppy/osu/assets/20418176/2f559da4-a734-4243-b815-b5dd22b9761e) | ![osu_2024-01-25_10-00-11](https://github.com/ppy/osu/assets/20418176/1cb3e60f-0e03-4ebc-8e9f-60497ba40d2d) |
| scrolled maximally to end | ![osu_2024-01-25_09-59-45](https://github.com/ppy/osu/assets/20418176/d37d3cc6-e1f9-404c-9b3b-d03a0bd34fa0) | ![osu_2024-01-25_10-00-18](https://github.com/ppy/osu/assets/20418176/33cafbae-d159-4e9d-b370-f05a6feb9d9e) |

Reply:
Yeah I noticed this changes things slightly, but I think it's okay? Can adjust if you think there's a reason to.

Reply:
It annoys me because not only does it look wrong and imbalanced but it also breaks right-click-to-absolute-scroll slightly. Like try right-clicking in the padded area at the bottom:

https://github.com/ppy/osu/assets/20418176/5562d6dd-d4d0-4ee4-8c42-5ada7b7bb5ab

I wanted to try and offer constructive solutions to the issue but I'm afraid that I've got none right now.

Reply:
I've noticed the right scroll was clamping from the top but not the bottom. This seems to fix it:

```diff
diff --git a/osu.Game/Graphics/Containers/OsuScrollContainer.cs b/osu.Game/Graphics/Containers/OsuScrollContainer.cs
index 124becc35a..ffd28957ef 100644
--- a/osu.Game/Graphics/Containers/OsuScrollContainer.cs
+++ b/osu.Game/Graphics/Containers/OsuScrollContainer.cs
@@ -127,7 +127,7 @@ protected override bool OnScroll(ScrollEvent e)
         }
 
         protected virtual void ScrollFromMouseEvent(MouseEvent e) =>
-            ScrollTo(Clamp(ToLocalSpace(e.ScreenSpaceMousePosition)[ScrollDim] / DrawSize[ScrollDim]) * Content.DrawSize[ScrollDim], true, DistanceDecayOnRightMouseScrollbar);
+            ScrollTo(Clamp(ToLocalSpace(e.ScreenSpaceMousePosition)[ScrollDim] / DrawSize[ScrollDim] * Content.DrawSize[ScrollDim]), true, DistanceDecayOnRightMouseScrollbar);
 
         protected override ScrollbarContainer CreateScrollbar(Direction direction) => new OsuScrollbar(direction);
 
```

https://github.com/ppy/osu/assets/35318437/d3877270-cd39-4306-a84d-b9412e5f96ff

Reply:
Seems to fix so I've applied.

> May want to change the position to be based on the middle of the scrollbar (not top).

I'm not holding up this PR further for that.